posthog/api/test/__snapshots__/test_insight.ambr

```diff
@@ -62,8 +62,8 @@
                        HAVING max(is_deleted) = 0) person ON person.id = pdi.person_id
                     WHERE team_id = 2
                       AND event IN ['user did things', 'user signed up']
-                      AND toDateTime(timestamp, 'UTC') >= toDateTime('2023-01-17 00:00:00', 'UTC')
-                      AND toDateTime(timestamp, 'UTC') <= toDateTime('2023-01-24 23:59:59', 'UTC')
+                      AND toDateTime(timestamp, 'UTC') >= toDateTime('2023-01-18 00:00:00', 'UTC')
```

Comment:
It's surprising that this snapshot change is in this PR

Reply:
I... don't know. It... [wasn't me](https://github.com/PostHog/posthog/pull/13918/commits/aa462a0c594742b1c7cc63350b059f39129ee858) 😬